src/app/faq/page.module.scss

```
@@ -1,3 +1,55 @@
+@use 'src/app/variables';
+@mixin box-shadow($x, $y, $blur, $color) {
+  -webkit-box-shadow: $x $y $blur $color;
+  -moz-box-shadow: $x $y $blur $color;
+  box-shadow: $x $y $blur $color;
+}
+
+.head {
+  text-decoration: underline;
+}
+
 .container {
-  padding-top: 72px;
-}
\ No newline at end of file
+  flex-direction: column;
+  background-color: variables.$background-color;
+}
+
+.contents {
+  margin: 40px;
+  padding: 40px;
+  background-color: #ffffff;
+}
+
+.QAcontainer {
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.chatMessageQuestion {
+  @include box-shadow(0px, 4px, 10px, rgba(0, 0, 0, 0.1));
+  width: fit-content;
+  padding: 1em; 
+  background-color: #f1f1f1;
+  border-radius: 10px;
+  .chara {
+    float: left;
+    font-size: 32px;
+    color: #439154;
+    margin: 0 40px 0 0;
+  }
```

この文字は `Q` 固定 なので`::before` 疑似要素を使ってもいいかもしれない (`A` も同様)

```suggestion
  &::before {
    content: 'Q';
    font-size: 2rem;
    color: #439154;
    margin: 0 1rem 0 0;
  }
```
(間違ってるかも)